File: sort/sort.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func bubleSort(array []int) []int {
	length := len(array)
	for i := 0; i < length; i++ {
		for j := 0; j < length-i-1; j++ {
			if array[j] > array[j+1] {
				array[j+1], array[j] = array[j], array[j+1]
			}
		}
	}
	return array
}
func main() {
	array := make([]int, 10000)
	for i := 0; i < len(array); i++ {
		array[i] = rand.Intn(1000)
	}
	startTime := time.Now()
	bubleSort(array)
	endTime := time.Since(startTime)
	fmt.Println(endTime)
}

File: sort/sort.php
<?php
function bubleSort($arr){
    $length = count($arr);
    for ($i = 0; $i < $length; $i++){
        for ($j = 0; $j < $length - $i - 1; $j++){
            if ($arr[$j] > $arr[$j + 1]){
                $tmp = $arr[$j];
                $arr[$j] = $arr[$j + 1];
                $arr[$j + 1] = $tmp;
            }
        }
    }
}
$arr = array();
for ($i = 0; $i < 10000; $i++){
    $arr[] = rand(1, 1000);
}
$start_time = hrtime(true);
bubleSort($arr);
$endtime = hrtime(true);
echo ($endtime - $start_time)/1e+6," ms\n";